iina/MPVController.swift

```diff
@@ -244,6 +250,20 @@ class MPVController: NSObject {
     // get version
     mpvVersion = getString(MPVProperty.mpvVersion)
   }
+  private func getFilePath(Configs userConfigs: [String: Any]!, forConfig conf: String, showAlert: Bool = true) -> String? {
```

Comment:
Please try to be DRY (Don't Repeat Yourself). 

We can move `getFilePath()` from `PrefKeyBindingViewController` to `Utility` as a static function.

(Also there's a typo: getFilePath(*for*Configs ...))